apps/web/tsconfig.json

```diff
@@ -15,7 +15,8 @@
       "incremental": true,
       "paths": {
         "@/*": ["./src/*"],
-        "@openstad/*": ["../../packages/*"]
+        "@openstad/*": ["../../packages/*"],
+        "react": [ "./node_modules/@types/react" ]
```

Comment:
Wat was de reden dat dit toegevoegd moest worden?

Reply:
De toevoeging bij de tsconfig was de uiteindelijke oplossing die ik heb gevonden om te zorgen dat de error bij Comp en dergelijke niet langer voorkwam. Van wat ik begrijp komt de error voor omdat de applicatie in bepaalde dependencies niet de juiste React types gebruikt ondanks dat de package.json hierop aangepast is. Met de toegevoegde regel forceer je in principe dat de applicatie overal de juiste types gebruikt.